ruby_event_store/ruby_event_store.gemspec

```diff
@@ -19,6 +19,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
+
+  spec.add_development_dependency 'activesupport'
```

Comment:
@andrzejsliwa it's a runtime dependency, not a development.

Reply:
ups, thx